### PR TITLE
updated instructions for setting up a validator

### PR DESCRIPTION
**Pull request by FidelVe**
From feedback obtained by users in the discord channels I have updated the instructions for setting up and registering a validator to make it more easier for new people

### CODE REVIEW

**Comment by eunsoo-icon** [comment]
For security reasons, it is recommended to separate the `node` and `P-Rep` keys.
`P-Rep` key is used to register a P-Rep and receive rewards.
`node` key is used for block consensus and is deployed on the node server. (./config/keystore.json)
A P-Rep must register the public key of its `node` key via `setPRepNodePublicKey` to vote on block consensus.

**Comment by FidelVe** [comment]
> For security reasons, it is recommended to separate the `node` and `P-Rep` keys. `P-Rep` key is used to register a P-Rep and receive rewards. `node` key is used for block consensus and is deployed on the node server. (./config/keystore.json) A P-Rep must register the public key of its `node` key via `setPRepNodePublicKey` to vote on block consensus.

Thank you for the feedback.

Ill rewrite it to reflect those recommendations.

**Comment by FidelVe** [comment]
@eunsoo-icon 

I just updated the instructions to add the recommendations of using one wallet for the validator and one wallet for the node.

please review it and let me know if is ok to publish

File: pages/operate-icon/how-to-run-a-validator-node.mdx [hunk 1]
```diff
@@ -1,12 +1,20 @@
 # How to run a Validator node
 
-### Prerequisites
+Registering and running a validator node requires a few steps to be followed.
 
-Install [Docker](https://www.docker.com/get-started/)
+* Create a wallet for the validator and register it on the chain as a validator by paying the registration fee.
+* Setup the node and wait for it to fully sync up. This node will be setup with a node wallet (not the same as validator wallet for security reasons) that will be used to sign blocks and consensus.
+* Register the node with the validator public key on the chain by calling `setPRepNodePublicKey` method.
```
Comment (eunsoo-icon) [inline pages/operate-icon/how-to-run-a-validator-node.mdx]: How about changing the sequence? Then users don't waste validator registration fees when users fail to set up a node.
1. create a wallet for the node and setup the node 
2. create a wallet for the validator and register on the chain
3. register the node public key

Comment (FidelVe) [reply pages/operate-icon/how-to-run-a-validator-node.mdx]: completely agree, thanks for pointing that out.

I'll fix it and then publish 👍

**Pull request merged**